sixtyfps_runtime/corelib/model.rs

```diff
@@ -462,56 +462,49 @@ impl Model for bool {
     }
 }
 
-/// Properties of type array in the .60 language are represented as
-/// an [`Option`] of an [`Rc`] of something implemented the [`Model`] trait
-#[derive(derive_more::Deref, derive_more::DerefMut, derive_more::From, derive_more::Into)]
-pub struct ModelHandle<T>(pub Option<Rc<dyn Model<Data = T>>>);
+/// Properties of type Array in the .60 language are represented as
+/// a `ModelRc` that implements the `Model` trait.
+#[derive(derive_more::Deref, derive_more::DerefMut)]
```

Comment:
We still need to get rid of Deref and DerefMut, but we can do that in another PR